pkgs/development/libraries/libaom/default.nix

```diff
@@ -16,6 +19,13 @@ stdenv.mkDerivation rec {
     yasm perl cmake pkg-config python3
   ];
 
+  buildInputs = lib.optionals enableButteraugli [
+    # libavif -> libaom -> libjxl -> graphviz -> gd -> libavif
+    # graphviz is only used for generating docs so it's fine to remove when building libaom
+    (libjxl.override { graphviz = null; })
```

Comment:
overriding packages to null is a very big anti pattern and when done wrong can cause all sorts of issues. Also this should be done in a higher level do make sure we don't mix two variants.
I would suggest to add a flag to libjxl to not build the doc instead.

Reply:
That makes sense